Wikibase/ruleset.xml

```diff
@@ -4,9 +4,21 @@
 		https://github.com/wikimedia/mediawiki-tools-codesniffer/blob/master/MediaWiki/ruleset.xml
 		-->
 	<rule ref="vendor/mediawiki/mediawiki-codesniffer/MediaWiki">
+		<!-- This sniff is way to rigorous about way to many details that need exceptions, e.g.
+			obvious function headers that don't need a PHPDoc block. -->
+		<exclude name="MediaWiki.Commenting.FunctionComment" />
+
+		<!-- We disagree with the idea of certain characters making comments "illegal" and blocking
+			patches from being merged. This behaves especially bad on commented out code. -->
+		<exclude name="MediaWiki.Commenting.IllegalSingleLineComment" />
```

Comment:
It appears you entirely misunderstood what I meant. Let's please discuss this in person.

I tried this sniff on the Wikibase code base and found that it reports so many details that are not errors and don't need fixing, it's a pain to deal with this somehow, either by adding tons and tons of exceptions, or by adding useless PHPDoc blocks that don't make the code better. What the `<exclude …>` here means is simply: don't impose this onto users of this rule set. If they want this, they can enable it with a single line in their local phpcs.xml.